app/views/gallery/index.html.erb

```diff
@@ -18,3 +18,44 @@
   should always be a multiple of 4, this will ensure the copy aligns to the 8px baseline
   grid while allowing more flexibility in setting type than multiples of 8.
 </p>
+
+<div class="textfield js-textfield" id="1">
+  <label class='textfield__label' for="name">Name</label>
+  <div id="name-hint" class='textfield__help'>Enter a full name here</div>
+  <input
+    type='text'
+    class='textfield__input'
+    id="name"
+    name="name"
+    value="Fred smith"
+    autoComplete="no"
+  />
+  <div id="name-error" class='textfield__error'>Here is an error</div>
+</div>
+
+<div class="textfield js-textfield" id="2">
+  <label class='textfield__label ' for="name">Name</label>
+  <div id="name-hint" class='textfield__help'>Enter a full name here</div>
+  <input
+    type='text'
+    class='textfield__input'
+    id="name"
+    name="name"
+    value="John brown"
+    autoComplete="no"
+  />
+</div>
+
+<div class="textfield js-textfield has-error" id="3">
+  <label class='textfield__label' for="name">Name</label>
+  <div id="name-hint" class='textfield__help'>Enter a full name here</div>
+  <input
+    type='text'
+    class='textfield__input'
+    id="name"
+    name="name"
+    value="Jane Doe"
+    autoComplete="no"
+  />
+  <div id="name-error" class='textfield__error'>Here is an error</div>
+</div>
```

Comment:
i was using this to test before that was merged and was getting the markup right first